addons/l10n_in/i18n/l10n_in.pot

```diff
@@ -273,7 +283,7 @@ msgstr ""
 
 #. module: l10n_in
 #: model_terms:ir.ui.view,arch_db:l10n_in.res_config_settings_view_form_inherit_l10n_in
-msgid "Enable HSN/SAC Validation"
+msgid "HSN/SAC Validation"
 msgstr ""
 
```

Comment:
This looks clean but then it won't be alphabetically sorted. Any new generated `.pot` files will have this translation down among with the "H" stuffs, and will need to be re-positioned everytime.

I suggest just keep it simple, let these lines be deleted and write the new one in the default place. It's not that much lines of code to add anyway